templates/zerver/emails/macros.source.html

```diff
@@ -0,0 +1,19 @@
+{% macro email_tag(email, text="") -%}
+    {% if text %}
+    <a href="mailto:{{ email }}">{{ text }}</a>
+    {% else %}
+    <a href="mailto:{{ email }}">{{ email }}</a>
+    {% endif %}
+{%- endmacro %}
```

Comment:
Could be simplified using the Jinja `default` filter.

```
{% macro email_tag(email, text) -%}
    <a href="mailto:{{ email }}">{{ text | default(email) }}</a>
{%- endmacro %}
```

(Note removal of `=""`, since `default` only replaces undefined.)